docs/certificate-specification.md

```diff
@@ -0,0 +1,112 @@
+# Certificate Specification
+
+This document includes the requirements for root, intermediate, and issued certificates.
```

Comment:
Are you intending these restrictions for all instances of Fulcio, or just the instance operated by the sigstore community?

Reply:
I was thinking for the public instance, but this could be tweaked slightly for all instances. The main differences would be an optional intermediate, and an SCT not always being embedded. I’d also want to exclude ephemeral, since that’s for testing.

is there a preference?

Reply:
I have a strong preference towards the issued x509 certificate restrictions being required for all instances -- the soundness of our verification semantics for depends on there being exactly one SAN, for example, and other instances should conform to that constraint.

Reply:
(But I agree re: intermediates and detached SCTs.)

Reply:
Agreed - I’ll make these updates.

Reply:
Updated!

Reply:
Updated!